src/redux/configureStore.js

```diff
@@ -0,0 +1,7 @@
+import { legacy_createStore as createStore, combineReducers } from 'redux';
```

Comment:
- [OPTIONAL] Import `applymiddleware` - to use some handy Redux middleware, like a redux logger.
  * it will help a lot in the development process. To install it, you need to run:

    ```
    npm i --save redux-logger
    ```